packages/transmute-services/test/Oracle/interface.truffle.js

```diff
@@ -1,20 +1,41 @@
-var OracleFactory = artifacts.require("./OracleFactory.sol");
 var Oracle = artifacts.require("./Oracle.sol");
 var OracleCaller = artifacts.require("./OracleCaller.sol");
 
 const chai = require("chai");
 const expect = chai.expect;
 const TransmuteFramework = require("transmute-framework").default;
 
-const listenForRequest = oracle => {
+const listenForRequest = (T, accounts, oracle, caller) => {
```

Comment:
nice.

Reply:
There's some updating to do here as well - the watcher needs to terminate what it is doing. I'll probably make this anonymous async given that we have one Oracle that is being tested here.